windowsforms/FlexGridShowcaseDemo/cs/MainForm.cs

```diff
@@ -0,0 +1,632 @@
+﻿using C1.Win.FlexGrid;
```

Comment:
There are a lot of unnecessary allocations in this file. It is quite likely users will be copying chunks of the sample code verbatim, and with that will be copying suboptimal implementations.

Reply:
I tried to resolve all unnecessary allocations.
Please, take a look into 3780348f25f9178612470f87dc383a199eee468e.